Frontend/CO.CDP.OrganisationApp/Pages/Shared/_FlashMessage.cshtml

```diff
@@ -35,6 +35,36 @@
         @{
             FlashMessage message = tempDataService.Get<FlashMessage>(FlashMessageTypes.Important)!;
         }
+        <div class="govuk-notification-banner__content">
+            <p class="govuk-notification-banner__heading">@Html.Raw(message.Heading)</p>
+
+            @if (message.Description != null)
+            {
+                <p>@Html.Raw(message.Description)</p>
+            }
+        </div>
+    </div>
+}
+
+@if (tempDataService.Peek<FlashMessage>(FlashMessageTypes.Failure) != null)
+{
+    FlashMessage message = tempDataService.Get<FlashMessage>(FlashMessageTypes.Failure)!;
+
+    <div class="govuk-notification-banner govuk-!-margin-top-8 app-notification-banner--failure" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
+        <div class="govuk-notification-banner__header">
+            <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+                @if (message.Title != null)
+                {
+                    @message.Title
+                    ;
+                }
+                else
+                {
+                    @:Failure
```

Comment:
```suggestion
                    @StaticTextResource.Global_Failure
```

Reply:
I know you've done this in your other PR (#1011)- if you want to leave this one and do it there that's fine - just so long as it comes out in the wash